From e649ac7f6d54c7ca078731bb3bcbaa9e7b40d01a Mon Sep 17 00:00:00 2001
From: Jason Wang <jasowang@redhat.com>
Date: Mon, 29 Sep 2014 00:59:16 -0400
Subject: [virt] hyperv: Adjust the size of sendbuf region to support ws2008r2

Message-id: <1411952358-34104-33-git-send-email-jasowang@redhat.com>
Patchwork-id: 96777
O-Subject: [RHEL7.1 PATCH 32/34] hyperv: Adjust the size of sendbuf region to support ws2008r2
Bugzilla: 1146357
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Vitaly Kuznetsov <vkuznets@redhat.com>

From: KY Srinivasan <kys@microsoft.com>

Bugzilla: 1146357

WS2008R2 is a supported platform and it turns out that the maximum sendbuf
size that ws2008R2 can support is only 15MB. Make the necessary
adjustment.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit be136ed30a4345e42ad846c1b9d925932dab303b)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/hyperv/hyperv_net.h b/drivers/net/hyperv/hyperv_net.h
index 8f0013a..4170708 100644
--- a/drivers/net/hyperv/hyperv_net.h
+++ b/drivers/net/hyperv/hyperv_net.h
@@ -586,7 +586,7 @@ struct nvsp_message {
 
 #define NETVSC_RECEIVE_BUFFER_SIZE  (1024*1024*16) /* 16MB */
 #define NETVSC_RECEIVE_BUFFER_SIZE_LEGACY (1024*1024*15)  /* 15MB */
-#define NETVSC_SEND_BUFFER_SIZE   (1024 * 1024 * 16)   /* 16MB */
+#define NETVSC_SEND_BUFFER_SIZE   (1024 * 1024 * 15)   /* 15MB */
 #define NETVSC_INVALID_INDEX   -1
 
 
-- 
1.7.1